build_tools/python/e2e_test_framework/device_specs/pixel_4_specs.py

```diff
@@ -27,3 +27,11 @@
     device_parameters=[device_parameters.ARM_LITTLE_CORES],
     tags=["little-core"],
 )
+ALL_CORES = common_definitions.DeviceSpec.build(
+    id=unique_ids.DEVICE_SPEC_MOBILE_PIXEL_4 + "-all-cores",
```

Comment:
nit: For consistency, maybe `-all-core`?

Reply:
Changed to `all-core`.